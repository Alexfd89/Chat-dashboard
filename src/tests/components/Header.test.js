import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render Header currectly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});