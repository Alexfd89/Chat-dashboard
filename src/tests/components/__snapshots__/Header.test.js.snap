// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header currectly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
</div>
`;
